java/src/main/proto/spine/validation/java/type.proto

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+syntax = "proto3";
+
+package spine.validation.test;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.validation.test";
+option java_outer_classname = "BlizzardProto";
+option java_multiple_files = true;
+option (internal_all) = true;
+
+import "spine/protodata/ast.proto";
+
+// A tuple of a message type and the file where it is declared.
+//
+message MessageWithFile {
+
+    spine.protodata.MessageType message = 1;
```

Comment:
@armiol, @dmdashenkov, similarly to Validation, we intended to market ProtoData as a relatively independent product.

What if we have proto packages starting from `protodata` (instead of `spine.protodata`)?

Reply:
This question does not relate to this PR, but I take the chance to discuss it here.

Reply:
@dmdashenkov @alexander-yevsyukov let's discuss it in a meeting.

My vote is for `spine.` removal here. And! Somewhen we'll need to come up with a name for the Validation library, so that we could drop the `spine.` prefix there as well.

Reply:
Our validation options which currently belong to `base`. They do not have a package for brevity of declarations at usage sites.  So, we do not have the longer prefix issue for these types at all.

Someday `options.proto` may become a part of the Validation library (thus making `base` depend on `validation`) for the sake of cohesion of the types. But it's not a priority now. I'm talking about this in the context of discussing types of the Validation library.